### PR TITLE
lint more but use .eslintignore

### PR DESCRIPTION
**Pull request by deepsweet**
because of `"lint": "eslint src/"` `.eslintignore` was useless

### CODE REVIEW

**Comment by deepsweet** [comment]
btw, why you are using `node --harmony --harmony_arrow_functions ./node_modules/tape/bin/tape test/*.js` for tests? why not babel-node?

**Comment by ghost** [comment]
> btw, why you are using node --harmony --harmony_arrow_functions ./node_modules/tape/bin/tape test/*.js for tests? why not babel-node?

I just wanted some harmony features, but didn't want to write the full spec in ES6 and add another transpilation step.

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-bin/*
-dist/*
-test/*
+bin/
+examples/
+dist/
```
Comment (ghost) [inline .eslintignore:2]: :+1:

File: .eslintignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
-bin/*
-dist/*
-test/*
+bin/
+examples/
```
Comment (ghost) [inline .eslintignore]: Why?

File: dist/cli/list.js [hunk 3]
```diff
@@ -13,14 +13,14 @@ exports.list = list;
 
 var _fmt = require("../fmt");
 
+var _fmt2 = _interopRequireDefault(_fmt);
```
Comment (ghost) [inline dist/cli/list.js:16]: What happened here?

Comment (deepsweet) [inline dist/cli/list.js:16]: looks like newer babel version than in your current dist, this can be ignored with my another PR

File: .eslintignore (hunk 2 above)
Comment (deepsweet) [inline .eslintignore]: I just wanted to lint `test/` folder, so `"lint": "eslint src/"` is now `"lint": "eslint ."` and we need to ignore some folders in .eslintignore, like `examples/`

**Comment by deepsweet** [comment]
anyway, it seems like currently existing `.eslintignore` file is not used because of `"lint": "eslint src/"`

File: .eslintignore (hunk 2 above)
Comment (ghost) [inline .eslintignore]: I think there is no prob with linting `examples` too.

**Comment by ghost** [comment]
Fixed in the latest release. Thanks!
